### PR TITLE
Support 'store_result' option in the JDBC operator family

### PR DESCRIPTION
**Pull request by komamitsu**

### CODE REVIEW

**Comment by komamitsu** [comment]
@frsyuki Can you review this pull request?

File: digdag-tests/src/test/java/acceptance/PgIT.java [hunk 1]
```diff
@@ -199,6 +199,73 @@ public void selectAndDownloadWithNullValues()
...
+        setupSourceTable();
+
+        CommandStatus status = TestUtils.main(
+                "run", "-o", root().toString(),
+                "--project", root().toString(),
+                "-p", "pg_database=" + tempDatabase,
+                "-p", "outfile=out",
+                "-X", "config.pg.max_store_result_rows=2",
+                "pg.dig");
+        assertThat(status.code(), is(1));
```
Comment (frsyuki) [inline digdag-tests/src/test/java/acceptance/PgIT.java]: It's better to check error message as well as exit code to confirm that tests don't wrongly success when pg operator failed to parse `config.pg.max_store_result_rows` option.

File: digdag-tests/src/test/java/acceptance/PgIT.java [hunk 2]
```diff
@@ -199,6 +199,73 @@ public void selectAndDownloadWithNullValues()
...
+
+        setupSourceTable();
+
+        CommandStatus status = TestUtils.main(
+                "run", "-o", root().toString(),
+                "--project", root().toString(),
+                "-p", "pg_database=" + tempDatabase,
+                "-p", "outfile=out",
+                "pg.dig");
+        assertThat(status.code(), is(0));
```
Comment (frsyuki) [inline digdag-tests/src/test/java/acceptance/PgIT.java]: Adding first message argument to assertThat is very helpful for debugging:

```
assertThat(status.errUtf8(), status.code(), is(0));
```

This shows `status.errrUtf8()` when the assertion fails. It informs reason of the failure which is helpful especially on CI.

File: digdag-tests/src/test/java/acceptance/PgIT.java (hunk 1 above)
Comment (komamitsu) [reply digdag-tests/src/test/java/acceptance/PgIT.java]: 👍 
Depending on an error message is a bit naive, but it's better than never. Maybe we'd better add error codes more reliable in the future.

**Comment by komamitsu** [comment]
> I think it's better to consider consistency with td> operator.

Sounds reasonable.
Addressed your feedbacks.

**Comment by komamitsu** [comment]
@frsyuki Updated. Can you take a look when you get a chance?

**Comment by frsyuki** [comment]
Nice 👍  LGTM

**Comment by komamitsu** [comment]
Thanks!

**Comment by frsyuki** [comment]
Following-up: #663.

**Pull request merged**